modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -63,22 +61,21 @@
 			<param index="0" name="what" type="Variant" />
 			<param index="1" name="type" type="int" />
 			<description>
-				Converts from a type to another in the best way possible. The [code]type[/code] parameter uses the [enum Variant.Type] values.
+				Converts [param what] to another [param type] in the best way possible. The [param type] uses the [enum Variant.Type] values.
```

Comment:
`another [param type]` doesn't make much sense, there's only single `type` parameter.

Either:
```suggestion
				Converts [param what] to another type in the best way possible. The [param type] uses the [enum Variant.Type] values.
```
or:
```suggestion
				Converts [param what] to [param type] in the best way possible. The [param type] uses the [enum Variant.Type] values.
```

Reply:
"Another" doesn't make much sense either, because it could really just be `convert(Vector2(), TYPE_VECTOR2)` without any issue, so I will stick the alternative in.